src/sentry/models/savedsearch.py

```diff
@@ -6,17 +6,44 @@
 from sentry.db.models import FlexibleForeignKey, Model, sane_repr
 
 
+DEFAULT_SAVED_SEARCHES = [
+    {
+        'name': 'Unresolved Issues',
+        'query': 'is:unresolved',
+        'is_default': True
+    },
+    {
+        'name': 'Needs Triage',
+        'query': 'is:unresolved is:unassigned'
+    },
+    {
+        'name': 'Assigned To Me',
+        'query': 'is:unresolved assigned:me'
+    },
+    {
+        'name': 'My Bookmarks',
+        'query': 'is:unresolved bookmarks:me'
+    },
+    {
+        'name': 'New Today',
+        'query': 'is:unresolved age:-24h'
+    },
+]
+DEFAULT_SAVED_SEARCH_QUERIES = set([s['query'] for s in DEFAULT_SAVED_SEARCHES])
+
+
 class SavedSearch(Model):
     """
     A saved search query.
     """
     __core__ = True
 
-    project = FlexibleForeignKey('sentry.Project')
+    project = FlexibleForeignKey('sentry.Project', null=True)
     name = models.CharField(max_length=128)
     query = models.TextField()
     date_added = models.DateTimeField(default=timezone.now)
     is_default = models.BooleanField(default=False)
+    is_global = models.BooleanField(default=False)
```

Comment:
iirc, per a conversation with @mattrobenolt a while ago, whenever we are adding columns to an existing table, they should be nullable and not have a default to ensure it won't acquire a lock / cause downtime

Reply:
Wasn't sure if it'd be necessary for this table since it's relatively small (1-2MM rows), but I guess the lock will still be held for a while. cc @mattrobenolt just to confirm

Reply:
Correct, plz to make this null and we can backfill the boolean.

Reply:
@mattrobenolt Will do, should I include that backfill in the data migration?